docs/cubeviz/import_data.rst

```diff
@@ -71,29 +78,38 @@ method, which takes as input a :class:`~specutils.Spectrum1D` object.
 
 FITS Files
 ----------
+
 The example below loads a FITS file into Cubeviz::
 
-    from specutils import Spectrum1D
-    spec3d = Spectrum1D.read("/path/to/data/file")
+    from jdaviz import Cubeviz
     cubeviz = Cubeviz()
-    cubeviz.load_data(spec3d)
+    cubeviz.load_data("/path/to/data/file.fits")
     cubeviz.show()
 
-You can also pass the path to a file that `~specutils.Spectrum1D` understands directly to the
+Spectrum1D (from file)
+----------------------
+
+For cases where the built-in parser is unable to understand your file format,
+you can try the `~specutils.Spectrum1D` parser directly and then pass the object to the
```

Comment:
are there cases where this happens or is it more just useful if the user wants to _modify_ the object before loading it?

Reply:
I dunno. It was an existing example. I just moved it into its own section. @orifox wrote it, I think.

Reply:
my question was about the added "For cases where the built-in parser is unable to understand your file format" part... are there cases where the parser would fail but specutils would not?

Reply:
Auto-guessing probably would fail for some edge cases, which then would require extra keywords into the specutils API that we don't support here, so yeah, I can see it happening but not often.